ethcontract-mock/src/details/default.rs

```diff
@@ -0,0 +1,25 @@
+//! Helpers for building default values for tokens.
+
+use ethcontract::common::abi::{Bytes, Int, ParamType, Token, Uint};
+use ethcontract::Address;
+
+/// Builds a default value for the given solidity type.
+pub fn default(ty: &ParamType) -> Token {
+    match ty {
+        ParamType::Address => Token::Address(Address::default()),
+        ParamType::Bytes => Token::Bytes(Bytes::default()),
+        ParamType::Int(_) => Token::Int(Int::default()),
+        ParamType::Uint(_) => Token::Uint(Uint::default()),
```

Comment:
Should we consider all the possible sizes of (u)int here like 32, 96, 128, ... the list is kinda big though).

Reply:
Not really, there's only one token for `Uint`, and that's a `U256`. All smaller ints are casted to it.

Reply:
Also note that the default is 0 which is the same regardless of the size.